tools/encode_program_information/encode_program_information.cpp

```diff
@@ -32,6 +32,20 @@ _emit_program_information_file(
     return EBPF_ERROR_SUCCESS;
 }
 
+static ebpf_helper_function_prototype_t _ebpf_helper_function_prototype[] = {
```

Comment:
In the future can we somehow have a way to ensure this array matches the definitions in ebpf_helpers.h ?
E.g., could that part of ebpf_helpers.h be dynamically generated from this, or generate both from an IDL or a .proto file?

Reply:
In the future, the helper function list returned will be dynamically composed. It will contain:
1) Helper functions from all the "global" helper function providers.
2) Helper functions from this program type provider (i.e. provider specific helper functions).

What helper functions are valid is a function of what extensions are loaded.